Tests/xcodeprojMeasurementsTests/XcodeProjGenerator.swift

```diff
@@ -0,0 +1,144 @@
+import Foundation
+import PathKit
+@testable import xcodeproj
+
+final class XcodeProjGenerator {
+    
+    struct GenerateSetting {
```

Comment:
These parameters allow us to generate a fairly large project - approximately 9 megabytes.

Reply:
This is a good idea. Thanks for bringing it into the project @CognitiveDisson